hr_emergency_contact/models/hr_employee.py

```diff
@@ -3,20 +3,17 @@
 # © 2016 OpenSynergy Indonesia
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from openerp import models, fields
+from odoo import models, fields
 
 
 class HrEmployee(models.Model):
 
     _inherit = 'hr.employee'
 
     emergency_contact_ids = fields.Many2many(
-        string='Emergency Contacts',
         comodel_name='res.partner',
-        relation='rel_employee_emergency_contact',
```

Comment:
Neither do i :smile: @feketemihai do you have opinion regarding this matter?

Well that's my old fashioned line of code :smile: . CMIIW, The reason i use it because on really-really rare occasion the creation of relationship table will raise an exception if you have 2 many2many fields on same object that have same comodel_name. Take a look at code on how odoo name the table https://github.com/odoo/odoo/blob/9.0/openerp/osv/fields.py#L947.